text/0000-roadmap-2021.md

```diff
@@ -0,0 +1,127 @@
+
+# Motivation
+[motivation]: #motivation
+
+The Rust project governance has grown into a large and diverse structure, with most teams not having well defined expectations about scope and membership. As the project has grown, we have also struggled with a unified place for new and existing contributors to learn about active efforts and join in. Our efforts to establish charters and unified vocabulary will help contributors new and old to navigate our organizational structure. Our expectation is that this work will also help each team better understand what their aims are and will help the core team better serve the team's needs.
+
+# Explanation
+[explanation]: #explanation
+
+## Chartering work in the Rust project
+
+The Rust teams, in concert with the core team, will work to establish a charter for each of the Rust teams over the course of the year, with an aim for defining, particularly, the purpose and membership requirements. Our goal is that going into 2022, all active groups within the Rust project will have well-defined charters and membership. This includes all pre-existing groups in Rust governance, including domain working groups, project groups, and working groups.
+
+We have a number of active efforts in the Rust project which are not currently called teams. By the end of the year, we expect that regardless of the naming, every active group in Rust’s governance will be chartered. The core team will additionally propose an RFC clarifying the naming and setting policy guidelines for team formation towards the end of the year, once we have worked through a number of charters and can set these expectations better.
+
+This RFC does not aim to provide a fine-grained framework for precisely what form these charters will take. Furthermore, it is our expectation that these charters will be “living” documents, parts of which will be updated as the group changes roles or membership, and the precise expectations of charters are likely to evolve over time.
+
+However, at minimum, they should provide answers to these questions:
+
+* What is expected of members to be part of this group?
+* Who is the point of contact for questions on the state of this group?
+* Where does this group work?
+* What does this group do?
+* How does this group make decisions?
+
+### What is expected of members to be part of this group?
+
+The membership expectations can be diverse for various groups. Some groups can meet multiple times a week and expect team members to devote time outside these meetings to drive proposals and otherwise contribute, while other teams can have no synchronous meetings and expect bursts of work from time to time.
+
+Especially when multiple teams are interfacing, or a new member is asked to join a team, having this documentation is immensely valuable to communicate up front the ground rules for involvement.
+
+### Who is the point of contact for questions on the state of this group?
```

Comment:
This is targeted at core team / other teams, who may want to e.g. implement a feature in a related area or otherwise get in contact to chat. New contributors should likely go for the #rustdoc channel or similar, I absolutely agree. I've added a paragraph to try and clarify that.